tests/test_invoice.py

```diff
@@ -11,24 +12,29 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.test.testcases import TestCase
-from mock import patch
+from mock import patch, ANY
 
-from djstripe.models import Customer, Invoice, Account
-from tests import FAKE_INVOICE, FAKE_CHARGE, FAKE_CUSTOMER, FAKE_SUBSCRIPTION
+from djstripe.models import Customer, Invoice, Account, UpcomingInvoice
+from djstripe.models import InvalidRequestError
+
+from tests import (FAKE_INVOICE, FAKE_CHARGE, FAKE_CUSTOMER,
```

Comment:
These imports are fine on two lines. Note that we exclude PEP8 E501 from our linting.